### PR TITLE
Added remote and help flag

### PR DESCRIPTION
**Pull request by hi2gage**
Added remote flag for better usability and help flag so that the user is able to see all of the fun things they can do with git-checkout-interactive-rust.

There might be a better way to handle arguments in `get_branches()` function.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -19,8 +33,13 @@ fn open_curr_repo() -> Repository {
     }
 }
 
-fn get_branches(repo: &Repository) -> Vec<(git2::Branch, git2::BranchType)> {
-    match repo.branches(Some(git2::BranchType::Local)) {
+fn get_branches(repo: &Repository, remote_branch: bool) -> Vec<(git2::Branch, git2::BranchType)> {
```
Comment (jiito) [inline src/main.rs]: Let's break this up into two functions `get_branches_local` and `get_branches_remote`. Then we can do the branching logic in the `main` function and avoid passing a boolean flag.

Comment (hi2gage) [reply src/main.rs]: Completed

**Review by jiito — APPROVED** [review]
I think we should eventually find some way to reduce the code reuse out of these two functions but make it so there is no boolean flag passed in. Possibly this is just a wrapper for the library function that passes in the `git2::BanchType` for the flag.

**Comment by jiito** [comment]
@hi2gage please leave a comment so that I know you have learned something from this experience.

**Pull request merged**